build/yamls/windows/containerd/conf/Run-AntreaAgent-Containerd.ps1

```diff
@@ -1,4 +1,5 @@
 $ErrorActionPreference = "Stop"
 $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
 $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+$env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/k/antrea/bin"
```

Comment:
If you plan to add "$mountPath/k/antrea/bin" into $env:Path, you can not use the absolute path of antrea-agent.exe, otherwise, you don't need to add it into the env:path ?

Reply:
I think we can use absolute path also despite the path being present in PATH. Please help me to understand why it would not work? What error it returns?
Other thing is that, we can remove using entire absolute path and use just `antrea-agent` in next line since path to it is already added.

Reply:
My point is if you don't plan to directly access files in path `$mountPath/k/antrea/bin` (without using absolute path), no need to add it into $env:Path

Reply:
Got it.
This change is included here because `Run-AntreaAgent-Containerd` is also shared by antrea-windows-containerd-with-ovs and there we need this path to be added as we have copied ovs exes inside `/k/antrea/bin`.  
Now antrea-agent.exe is also in the same path and that path is added to env var PATH, so we can directly access antrea-agent.exe also without giving absolute path, but it was done for ovs exes, as a side effect it is done now for antrea-agent.exe as well.

Reply:
If you want to add this path to enable direct access to agent.exe, then why continue using $mountPath/k/antrea/bin/antrea-agent.exe to start the agent? I suggest either removing this path or updating the agent start command for both YAML files. @Atish-iaf

Reply:
updated both yaml files.

Reply:
@Atish-iaf 
if path $mountPath/k/antrea/bin is added to env then all bins  on this pathwill be directly accessible without absolute path , so lets directly use antrea- agent.exe as well

Reply:
it is done.